src/main/java/org/example/hospital/controller/DoctorMessageController.java

```diff
@@ -0,0 +1,39 @@
+package org.example.hospital.controller;
+
+import org.example.hospital.dto.MessageDTO;
+import org.example.hospital.service.MessageService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+@RestController
+@RequestMapping("/doctor/{doctorId}/message")
+public class DoctorMessageController {
+    private final MessageService messageService;
+
+    @Autowired
+    public DoctorMessageController(MessageService messageService) {
+        this.messageService = messageService;
+    }
+
+    @RequestMapping(value = "/send")
+    public ResponseEntity<String> sendMessage(@PathVariable Long doctorId, @RequestBody MessageDTO messageDTO){
+        messageService.addMessage(doctorId, messageDTO);
```

Comment:
Because you don't have login, just add hardcoded patientId as senderId
Long senderId = 3L;